package exceptionhandling;

public class Throw {
static void call() {
	throw new ArithmeticException("error");
}
	public static void main(String[] args) {
	call();

	}

}
